api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -748,6 +751,18 @@ public User setUserActivationKey(User user) throws MessageException {
 		return user;
 	}
 
+	/**
+	 * Delete this method when deleting {@link OpenmrsConstants#GP_HOST_URL}
+	 */
+	private static void copyHostURLGlobalPropertyToPasswordResetGlobalProperty() {
+		AdministrationService adminService = Context.getAdministrationService();
```

Comment:
Is there any reason why you did not just pass the adminService as a parameter in this method?

Reply:
Did you intentionally make this method static?

Reply:
> Is there any reason why you did not just pass the adminService as a parameter in this method?

I did not pass it as a parameter to decouple the code but I think passing it as a parameter is better. So I changed it according to your comment.



> Did you intentionally make this method static?

It was a mistake.